src/styles/styles.js

```diff
@@ -1491,6 +1491,19 @@ const styles = {
         flex: 1,
     },
 
+    chatItemDraft: {
+        display: 'flex',
+        flexDirection: 'row',
```

Comment:
I only see a single `...flex.flerRow` usage and lots of `fledDirection: 'row'` and it feels like anti patters. Can you link to the style guide for this? Spreading a single value doesn't give anything here, but it's even harder to type.

Reply:
I feel that it's more about the usage of `styles.flex1` instead of defining your own `myStyle` where you have `{ flex: 1 }`, but not about building a style object out of other objects completely.